extensions/amp-consent/amp-consent.md

```diff
@@ -369,6 +369,8 @@ following value scheme `on="event:idOfAmpConsentElement.accept/reject/dismiss"`
 
 - `dismiss`: instruct AMP to cancel `buildCallback` of components waiting for the consent, and hides the prompt UI.
 
+**_Note:_** The consent state value may be confusing in opt-in vs opt-out scenarios. The consent state value can be thought of as a signal to tell amp runtime to unblock components. It is up to you to interpret the `consentStateValue` along with the `consentString`. For example, in an opt-out scenario, the consent value might be `unknown`, which allows personalized ads to be served. However, in the opt-in scenario, `unknown` would trigger non-personalized ads to be served/ads to be blocked.
```

Comment:
`unknown` may not be a good example here, because from vendors and also `<amp-consent>`'s perspective, they don't know what is the current scenario and it's not their responsibility to interpret `unknown`. 

The consent state value is different from actions. I'd prefer us adding the comment here
https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/amp-consent.md#response

What about something like
"The consent state value `accepted` instructs AMP that the user gives consent (or doesn't revoke consent in opt-out case). This can be used to serve personalized ads in some scenarios. `rejected` means the user doesn't give consent, and vendors may decide not to serve personalized ad in such scenarios"

cc @jeffjose

Reply:
Ok, I will move to the response description. Also, I like the edit, it is cleaner.